packages/jest-haste-map/src/crawlers/node.ts

```diff
@@ -74,27 +74,22 @@ function find(
         callback(result);
         return;
       }
-      // node < v10.10 does not support the withFileTypes option, and
```

Comment:
there's another node 10 thing below we could simplify, but I'm not 100% sure what the correct solution is there (https://github.com/facebook/jest/pull/9514/files/9f0aae7c79090cc5275aa7b91e2bb5a02d2c36f0#r800046825)